packages/mephisto-review-hook/src/index.js

```diff
@@ -21,8 +22,8 @@ function useMephistoReview({
       return;
     }
     const DATA_URL = taskId
-      ? `/data/${taskId}`
-      : `/data?${page ? "page=" + page : ""}${
+      ? `${hostname}/data/${taskId}`
+      : `${hostname}/data?${page ? "page=" + page : ""}${
```

Comment:
allows you to specify where the review server exists. good for local development.